__tests__/app.test.js

```diff
@@ -142,23 +146,65 @@ describe("GET /api/articles/:article_id", () => {
                 expect(res.body.article).toHaveProperty("created_at")
                 expect(res.body.article).toHaveProperty("votes")
                 expect(res.body.article).toHaveProperty("article_img_url")
-              })
-            })
-    })
-    it("404: Not Found when article doesnt exist", () => {
-        return request(app)
+            });
+    });
+});
+it("404: Not Found when article doesnt exist", () => {
+    return request(app)
         .get("/api/articles/9999")
         .expect(404)
         .then((res) => {
             expect(res.body.msg).toBe("Not Found")
-        })
-    })
-    it("400: Bad Request when id is invalid", () => {
-        return request(app)
+        });
+});
+it("400: Bad Request when id is invalid", () => {
+    return request(app)
         .get("/api/articles/hello")
         .expect(400)
         .then((res) => {
             expect(res.body.msg).toBe("Bad Request")
-        })
-    })
+        });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
```

Comment:
Nice test! It'd be better if you also check that the article_id of each of these comments is 1